data/CMakeLists.txt

```diff
@@ -101,5 +101,5 @@ set(ALL_DATA_FILE
 )
 install(
   FILES ${ALL_DATA_FILE}
-  DESTINATION ${DATADIR}
+  DESTINATION ${PROJ_LIB_PATH}
```

Comment:
... but this looks strange.

Reply:
it's after [`PROJ_LIB`](https://proj.org/usage/environmentvars.html#envvar-PROJ_LIB)

Reply:
The naming isn't straightforward. In the context of cmake, this implies the library install path I think, though we know what `PROJ_LIB` means outside of cmake (the link above was what I was thinking of when I named it). What's the best name for this? `DATADIR` isn't too clear either